klippy/toolhead.py

```diff
@@ -289,13 +290,14 @@ def _advance_flush_time(self, flush_time):
         for sg in self.step_generators:
             sg(sg_flush_time)
         self.last_sg_flush_time = sg_flush_time
+        clear_history_time = self.reactor.monotonic() - MOVE_HISTORY_EXPIRE
```

Comment:
The trapq times are relative to the "print_time", not the "system time".  To go from a system time to a print_time use `self.mcu.estimated_print_time(systime)`.

Reply:
Done, I made everything a print time including the time provided to `flush_moves` and convert using `print_time_to_clock` rather than `get_clock`.